proto/lbm/token/v1/tx.proto

```diff
@@ -11,145 +11,172 @@ option (gogoproto.goproto_getters_all) = false;
 
-  // Approve allows one to transfer tokens on behalf of the approver
-  rpc Approve(MsgApprove) returns (MsgApproveResponse);
+  // AuthorizeOperator allows one to send tokens on behalf of the approver.
+  rpc AuthorizeOperator(MsgAuthorizeOperator) returns (MsgAuthorizeOperatorResponse);
 
-  // Issue defines a method to issue a token
+  // RevokeOperator revoke the authorization of the proxy to send the approver's tokens.
+  // Since: finschia
+  rpc RevokeOperator(MsgRevokeOperator) returns (MsgRevokeOperatorResponse);
+
+  // Issue defines a method to create a class of token.
   rpc Issue(MsgIssue) returns (MsgIssueResponse);
 
-  // Grant allows one to mint or burn tokens or modify a token metadata
+  // Grant allows one to mint or burn tokens or modify a token metadata.
   rpc Grant(MsgGrant) returns (MsgGrantResponse);
 
-  // Revoke revokes the grant
-  rpc Revoke(MsgRevoke) returns (MsgRevokeResponse);
+  // Abandon abandons a permission.
+  rpc Abandon(MsgAbandon) returns (MsgAbandonResponse);
 
-  // Mint defines a method to mint tokens
+  // Mint defines a method to mint tokens.
   rpc Mint(MsgMint) returns (MsgMintResponse);
 
-  // Burn defines a method to burn tokens
+  // Burn defines a method to burn tokens.
   rpc Burn(MsgBurn) returns (MsgBurnResponse);
 
-  // BurnFrom defines a method to burn tokens
-  rpc BurnFrom(MsgBurnFrom) returns (MsgBurnFromResponse);
+  // OperatorBurn defines a method to burn tokens.
+  rpc OperatorBurn(MsgOperatorBurn) returns (MsgOperatorBurnResponse);
 
-  // Modify defines a method to modify a token metadata
+  // Modify defines a method to modify a token class.
   rpc Modify(MsgModify) returns (MsgModifyResponse);
 }
 
-// MsgTransfer represents a message to transfer tokens from one account to another.
-message MsgTransfer {
-  string class_id = 1;
-  string from     = 2;
-  string to       = 3;
-  string amount   = 4 [(gogoproto.customtype) = "github.com/line/lbm-sdk/types.Int", (gogoproto.nullable) = false];
+// MsgSend defines the Msg/Send request type.
+message MsgSend {
+  string contract_id = 1;
+  string from        = 2;
+  string to          = 3;
+  string amount      = 4 [(gogoproto.customtype) = "github.com/line/lbm-sdk/types.Int", (gogoproto.nullable) = false];
 }
 
-// MsgTransferResponse defines the Msg/Transfer response type.
-message MsgTransferResponse {}
+// MsgSendResponse defines the Msg/Send response type.
+message MsgSendResponse {}
 
-// MsgTransferFrom represents a message to transfer tokens from one account to another by the proxy.
-message MsgTransferFrom {
-  string class_id = 1;
-  string proxy    = 2;
-  string from     = 3;
-  string to       = 4;
-  string amount   = 5 [(gogoproto.customtype) = "github.com/line/lbm-sdk/types.Int", (gogoproto.nullable) = false];
+// MsgOperatorSend defines the Msg/OperatorSend request type.
+message MsgOperatorSend {
+  string contract_id = 1;
+  string proxy       = 2;
```

Comment:
The message name is using the work "operator", but the field name is using the word "proxy".
It's better to unify the word.

Reply:
Because I cannot change the field name, I think I should preserve `MsgTransferFrom`. The term `operator` would be used only in the new interface (new queries, new transactions, and new events).